### PR TITLE
interfaces: add read access to /proc/cgroups and /proc/sys/vm/swappiness to system-observe

### PR DESCRIPTION
**Pull request by Mehdi-Bendriss**
This PR is based on the conversation [here](https://forum.snapcraft.io/t/auto-connect-request-for-opensearch-snap/31746/6) 

It adds read access to `/proc/cgroups` and `/proc/sys/vm/swappiness` to the `system_observe` interface

### CODE REVIEW

**Review by mardy — APPROVED** [review]
LGTM, thanks!

(but please wait for the security review before merging)

**Review by alexmurray — APPROVED** [review]
LGTM!

**Pull request merged**
